gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterUtils.java

```diff
@@ -112,6 +116,26 @@ public static Path getJobStateFilePath(boolean usingStateStore, Path appWorkPath
     return jobStateFilePath;
   }
 
+  /**
+   * Return the system properties from the input {@link Config} instance
```

Comment:
1) The method is not returning anything.
2) Please describe the special treatment of java.io.tmpdir/TMP_DIR.YARN_CACHE config

Reply:
Addressed.